hnn_core/network_builder.py

```diff
@@ -62,8 +63,10 @@ def _simulate_single_trial(neuron_net, trial_idx):
     h.finitialize()
 
     def simulation_time():
-        print('Simulation time: {0} ms...'.format(round(h.t, 2)))
+        pbar.update(10)
 
+    pbar = tqdm(total=neuron_net.net._params['tstop'], unit='ms',
```

Comment:
does this actually work on Jupyter notebook? I seem to remember they have a separate function for that. Or has that changed?

Reply:
They have a `gui` parameter I haven't tested yet, but the attached screenshot is from a jupyter lab without any modification